prettier/examples/prettier-format-from-gitrepo.yaml

```diff
@@ -0,0 +1,54 @@
+kind: PersistentVolumeClaim
```

Comment:
If you move this file from `examples` -> `tests` this will also double as e2e tests! :D